inc/save.php

```diff
@@ -116,8 +116,8 @@ function validate_line( $line, $line_number ) {
 					'type'    => 'warning',
 					'message' => sprintf(
 							/* translators: %s: Subdomain */
-							__( '"%s" does not appear to be a valid subdomain', 'ads-txt' ),
-							esc_html( $subdomain )
+							esc_html__( '"%s" does not appear to be a valid subdomain', 'ads-txt' ),
```

Comment:
This escapes the double quotes. My preference would actually be to have the placeholder wrapped in code tags but settled on this compromise because of the JS templating. Open to other ideas for how to separate the placeholder (in this case the domain) from the rest of the error message.

Reply:
same: maybe add the escaping around `$domain`?